README.md

```diff
@@ -8,6 +8,9 @@
 [![ease of contribution: medium](https://img.shields.io/badge/ease%20of%20contribution:%20medium-f4900c)](https://github.com/openclimatefix#how-easy-is-it-to-get-involved)
 [![Test Docker image](https://github.com/openclimatefix/uk-pv-national-gsp-api/actions/workflows/test-docker.yaml/badge.svg)](https://github.com/openclimatefix/uk-pv-national-gsp-api/actions/workflows/test-docker.yaml)
 
+> [!WARNING]
+> This API will soon be deprecated in favour of [quartz-api](https://github.com/openclimatefix/quartz-api). Please start planning your migration if you depend on this service.
+
```

Comment:
Would you be able to remove `Please start planning your migration if you depend on this service.` 
We are just trying to keep it really simple and high level. 
Thanks